src/components/drive-container.tsx

```diff
@@ -129,20 +129,37 @@ export default function DriveContainer(props: DriveContainerProps) {
             addFile(file, decodedDriveID, undefined, targetId);
         }
     };
+
     return (
-        <DriveView
-            key="local"
-            type="LOCAL_DRIVE"
-            name={'My Local Drives'}
-            onItemClick={onItemClick}
-            onItemOptionsClick={onItemOptionsClick}
-            onSubmitInput={item => onSubmitInput(item)}
-            onCancelInput={cancelInputHandler}
-            onDragStart={onDragStartHandler}
-            onDragEnd={onDragEndHandler}
-            onDropEvent={onDropEventHandler}
-            onDropActivate={onDropActivateHandler}
-            disableHighlightStyles={disableHoverStyles}
-        />
+        <>
```

Comment:
did you leave this commented out here on purpose?

Reply:
I'll hardcode some drives for the demo and then I'll cleanup before merging this pR